trigger_circleci_pipeline.sh

```diff
@@ -7,38 +7,42 @@ CIRCLECI_PROJECT_API_BASE="${CIRCLECI_API_BASE}/project/github/${GITHUB_REPOSITO
 CIRCLE_WORKFLOW_URL_BASE="https://circleci.com/workflow-run"
 
 branch="${BRANCH_REF#refs/heads/}"
-workflow_to_trigger=$1
```

Comment:
Want to add validation that this env var is not missing? And what about the kwargs var?

Reply:
Gave the kwargs var a default `{}`.